### DISCUSSION TITLE
Speed limited in vase mode...What is doing it???

### DISCUSSION THREAD

**Discussion by PhilBaz** [smartavionics/Cura#155] [Q&A]
Can't get over a medium speed in vase mode... 

I'm trying to control it with outer wall speed, I believe that's correct, and minimum layer time is set to 0. What's stopping me? I just hit a wall at some point. can't slice it to go any faster.

**Comment by 5axes** [comment]
Minimum layer time parameter ?

**Comment by smartavionics** [comment]
Please provide a project file that shows the problem, thanks.

**Reply by PhilBaz** [reply]
Hummm looking at the speed legend (in preview:speed) it looks like the max speed is in fact changing, it's the time estimate that is not changing. 
[vase mode speed limit.zip](https://github.com/smartavionics/Cura/files/7964424/vase.mode.speed.limit.zip)

**Comment by smartavionics** [comment]
Yes, the gcode shows a fast speed being used but for some reason the time estimate doesn't want to go less than 15.

**Comment by smartavionics** [comment]
Yes, the gcode shows a fast speed being used but for some reason the time estimate doesn't want to go less than 15.

**Reply by PhilBaz** [reply]
Found this on Klipper discord.... sounds about right
![khkj](https://user-images.githubusercontent.com/21558241/151675945-722551d0-7354-430f-a0fa-13ca14b2989c.jpg)
.

**Comment by smartavionics** [comment]
I think it's the low value of Default X/Y Jerk that is having an effect here. Increasing that setting allows the time estimate to reduce.

**Reply by PhilBaz** [reply]
Nice, thank you! I've downloaded the plugin so ill up the jerk to get accurate ETAs. Built a Voron so want to run this puppy fast :)  Thanks again for everything!
